service/client/client.js

```diff
@@ -619,6 +621,15 @@ export default class Client {
         );
     };
 
+    // Command routes
+    executeCommand = async (teamId, command, commandArgs) => {
```

Comment:
yes I was using this for a test that I ended up removing, will remove this for now